froide/settings.py

```diff
@@ -566,6 +566,8 @@ def is_pkce_required(client_id):
             "about": "/about/",
             "help": "/help/",
             "throttled": "/help/",
+            # TODO english?
+            "help_postupload_redaction": "/hilfe/plain/funktionen-der-plattform/schwaerzungen-durchfuehren/",
```

Comment:
this should go into the theme repo: https://github.com/okfde/fragdenstaat_de/blob/40c44c2188808c5feb1d49dbfde0093b34e435e8/fragdenstaat_de/settings/base.py#L719

Reply:
it's there: https://github.com/okfde/fragdenstaat_de/blob/bikubi/postupload-wip3/fragdenstaat_de/settings/base.py#L726

but since post-upload.vue is part of froide, and this help url is kind of expected (short of hard coded), isn't it weird to leave it undefined, then? or should the text which contains this link be also moved to the theme repo?